channels/views_test.py

```diff
@@ -722,6 +731,18 @@ def test_add_moderator(client, use_betamax, praw_settings, staff_jwt_header):
     assert resp.json() == {'moderator_name': 'othermoderator'}
 
 
+def test_add_subscriber(client, use_betamax, praw_settings, staff_jwt_header):
+    """
+    Adds a subscriber to a channel
+    """
+    client.force_login(UserFactory.create(username='fooadmin'))
```

Comment:
Shouldn't need to force a login since the `staff_jwt_header` is what we'll end up using to authenticate the user in normal cases. This is basically providing 2 auth mechanisms in this test. The JWT should take precedence anyway, but I wouldn't want us to have a false positive in case something with the JWT breaks.